config/includes.chroot/etc/initramfs-tools/scripts/init-premount/loop_mount

```diff
@@ -95,25 +112,25 @@ if [ "x${LOOPROOT}" != "x" ]; then
 		exit 1
 	fi
 
-	log_begin_msg "setting up loop device ${lprootd} from ${loopfile}"
+	_log_msg "Setting up loop device ${lprootd} from ${loopfile} ...\n"
 	losetup ${lprootd} ${loopfile} || {
 		rc=$?
 		log_failure_msg "losetup failed ${rc}"
 		exit ${rc}
 	}
-	log_end_msg
+	_log_msg "... losetup done\n"
 
-	log_begin_msg "probing loop device for partitions"
+	_log_msg "Probing loop device for partitions ...\n"
 	kpartx -a ${lprootd} || {
 		rc=$?
 		log_failure_msg "kpartx failed ${rc}"
 		exit ${rc}
 	}
-	log_end_msg
+	_log_msg "... probing done\n"
 
-	_log_msg "recording ntfs-3g driver in no-kill list"
+	_log_msg "Recording ntfs-3g driver in no-kill list\n"
 	mkdir -p /run/sendsigs.omit.d
-	pidof mount.ntfs-3g >> /run/sendsigs.omit.d/ntfs-3g
+	pidof mount.ntfs-3g >> /run/sendsigs.omit.d/ntfs-3g || true
```

Comment:
If there is no process named ``mount.ntfs-3g`` then ``pidof`` will fail with code 1 (so if looproot is anything other than ntfs it will fail)